lib/project_types/script/commands/push.rb

```diff
@@ -10,7 +10,8 @@ class Push < ShopifyCLI::Command::SubCommand
       end
 
       def call(_args, _name)
-        fresh_env = Tasks::EnsureEnv.call(@ctx)
+        fresh_env = Layers::Application::ConnectApp.call(ctx:@ctx)
+
         force = options.flags.key?(:force) || !!fresh_env
```

Comment:
This is looking a lot better!!!  Line 15-18 should be part of `Application::PushScript`, but that's outside the scope of this PR, and we can fix that up later, now that you know how to structure the code here.  👍